arviz/plots/autocorrplot.py

```diff
@@ -88,7 +95,7 @@ def plot_autocorr(
     .. plot::
         :context: close-figs
 
-        >>> az.plot_autocorr(data, var_names=['mu', 'tau'], combined=True)
+        >>> az.plot_autocorr(data, var_names=['~theta'], filter="like", combined=True)
```

Comment:
The tittle should be different like     "Combine chains collapsing by variable and negate variable names".
setting `filter="like"`is not necessary to negate a variable, right?

Reply:
- You're right about the title, will do.
- Indeed, it `filter` is not necessary to negate, it was just to show an example of how to use it, to vary examples. I will use "thet" instead of the full variable name, to make `filter` necessary.